magento/pwa-studio
# backmerge main to develop

# ❗ DO NOT SQUASH MERGE THIS PR ❗ 


## Description

Merge branch main to develop

## Related Issue







Closes [PWA-2758](https://jira.corp.magento.com/browse/PWA-2758)

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps





1. Verify regression fixes are in develop
2. All CI build should be green including cypress. (Ignore cypress filter test failure)

## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
